lib/helpers.js

```diff
@@ -323,4 +336,90 @@ helpers.isNestable = function (currentVal, previousVal, elements, nestable) {
   return false;
 };
 
+/**
+ * Tries to traverse the AST, following a specified path
+ * @param   {ASTNode} node           Starting node
+ * @param   {Array}   traversalPath  Array of Node types to traverse, starting from the first element
+ * @returns {Array}                  Nodes at the end of the path. Empty array if the traversal failed
+ */
+helpers.attemptTraversal = function (node, traversalPath) {
+  var i, nextNodeList, currentNodeList = [],
```

Comment:
It is for this reason that I prefer one var per declaration. Maybe we should write our own eslint rule to check for this.